src/components/RespondentsTable/RespondentsTable.tsx

```diff
@@ -84,6 +85,18 @@ export const RespondentsTable = (): JSX.Element => {
         filterRows(secretId, searchValue) || filterRows(nickname, searchValue),
     );
 
+  useEffect(() => {
+    if (id && baseBreadcrumbs && baseBreadcrumbs.length > 0) {
```

Comment:
Can we use just `(id && baseBreadcrumbs?.length) `here?

Reply:
done